metro.config.js

```diff
@@ -0,0 +1,18 @@
+// expo v41:
```

Comment:
Can you also explain what this file does, too?

Reply:
this is a config file for the metro bundler, which is used by expo. `react-native-svg-transformer` requires this configuration. I got it from here: https://github.com/kristerkari/react-native-svg-transformer#step-3-configure-the-react-native-packager